ksqldb-engine/src/test/java/io/confluent/ksql/structured/SchemaKTableTest.java

```diff
@@ -15,24 +15,22 @@
 
 package io.confluent.ksql.structured;
 
-import static org.easymock.EasyMock.anyObject;
```

Comment:
Not sure why we had both easymock and mockito in this test file but easymock is gone now 💀